telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java

```diff
@@ -15,7 +17,7 @@
   @Inject Analytics analytics;
 
   @Override public void onCreate() {
-    AndroidInjection.inject(this);
+    ((TelecineApplication) getApplication()).telecineComponent().api24OrGreaterServiceComponent().injector().inject(this);
```

Comment:
this is unfortunate. mind if i try something to see if we can avoid the Service knowing about its injector?

Reply:
opened #172 to keep the usefulness of the Android injection pieces.

Reply:
Cool. I got you the point. And your solution is much cleaner. I have also changed the implementation in my own project. Thank you for the help.

Reply:
Thank you for looking into it! This was a fun problem.